packages/whale-api-client/src/api/MasterNodes.ts

```diff
@@ -67,5 +67,6 @@ export enum MasternodeState {
   RESIGNED = 'RESIGNED',
   PRE_BANNED = 'PRE_BANNED',
   BANNED = 'BANNED',
-  UNKNOWN = 'UNKNOWN'
+  UNKNOWN = 'UNKNOWN',
+  TRANSFERRING = 'TRANSFERRING'
```

Comment:
If ownerAddress is updated on a masternode, the masternode will have `Transferring` state initially before moving to PRE_ENABLED state after several blocks and then, moving to ENABLED state after several blocks later too.